db/seeds.rb

```diff
@@ -16,7 +16,7 @@
     "Chair Representative" ],
   [ "hasso.plattner@hpi.de", "Hasso", "Plattner", "Prof. Dr. h.c.", "https://via.placeholder.com/150", "Professor" ],
   [ "mr.net@hpi.de", "Mr.", "Net", "", "https://via.placeholder.com/150", "" ],
-  [ "morpheus@student.hpi.de", "Morpheus", "Cyrani", "", "https://via.placeholder.com/150", "Tutor" ],
+  [ "morpheus@student.hpi.de", "Morpheus", "Cyrani", "Käpten zur See", "https://via.placeholder.com/150", "Tutor" ],
```

Comment:
So noch nicht ganz zufriedenstellend.
```suggestion
  [ "morpheus@student.hpi.de", "Morpheus", "Cyrani", "Käpten zur See", "https://i.ytimg.com/vi/HIFNsd5ayzU/hqdefault.jpg", "Tutor" ],
```